collections/_release-notes/2025-09-03-1-215-0.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 09.03.2025
+layout: productupdates
+date: 2025-09-03
+tags: sdk bugfix
+---
+
+Today's release includes the following updates:
+
+- Adds the Goal model to the SDK data module. [Read more](/sdk/data-goal)
+- Increases allowed characters for meeting_link column in appointments (to 1024 characters)
+- Adds “converted” state to Note object editable method. 
```

Comment:
@aduane would you be able to add a line in the docs for this (you can use this branch)